Docker/default.nix

```diff
@@ -28,12 +28,28 @@ let
       tar -C $out/ -xzf $src
     '';
   };
+  Borg = stdenv.mkDerivation {
+    name = "BorgSource";
+   
+    src = builtins.fetchGit {
+      rev = "2c7702638d42349824e305036fc6eb4a04a8a539";
+      url = "git@bitbucket.org:dmh309/serial-borg-moea.git";
+      ref = "master";
+    };
+  phases = "installPhase"; 
+ 
+  installPhase = ''  
+    mkdir -p $out/
+    cp -r $src/. $out/
+  '';
+  };     
 in
 stdenv.mkDerivation {
   name = "impurePythonEnv";
   buildInputs = [
     nix
     bash
+    git 
```

Comment:
Is it needed to have this here and in `dev-env.nix`? I would think just `dev-env.nix` would work, if I'm remembering right. Not a big deal, but worth a shot.

Reply:
Yeah, this is not needed, I have removed it in the most recent version.